### PR TITLE
feat(chart): Add OrdinalQuartile.ts

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by jbrundage** [comment]
image

**Comment by GordonSmith** [comment]
You should really have a common base class between this and the candlestick viz - "QuartileWidget" one which does all the math.  Then have the two visualizations derive from it and do the rendering:
CandlestickQuartile
BarQuartile

**Comment by jbrundage** [comment]
> You should really have a common base class between this and the candlestick

Agreed, I'll put this one on the back burner for now

File: packages/chart/src/OrdinalQuartile.ts [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+OrdinalQuartile.prototype.publish("valueFontSize", 12, "number", "Font size of value text (pixels)");
+OrdinalQuartile.prototype.publish("labelFontSize", 12, "number", "Font size of label text (pixels)");
+OrdinalQuartile.prototype.publish("roundedCorners", 8, "number", "Radius of rounded rect corners (pixels)");
+OrdinalQuartile.prototype.publish("lineWidth", 4, "number", "Width of lines (pixels)");
+OrdinalQuartile.prototype.publish("dataHeight", 60, "number", "Height of outer rect (pixels)");
+OrdinalQuartile.prototype.publish("textPadding", 8, "number", "Padding between outer rect and text (pixels)");
+OrdinalQuartile.prototype.publish("edgePadding", 30, "number", "Padding between outer rect and left/right sides (pixels)");
+OrdinalQuartile.prototype.publish("lineColor", "#333", "html-color", "Color of outer rect and lines");
+OrdinalQuartile.prototype.publish("labelTextRotation", -60, "number", "Rotation of label text (degrees)");
+OrdinalQuartile.prototype.publish("valueTextRotation", -60, "number", "Rotation of value text (degrees)");
```
Comment (GordonSmith) [inline packages/chart/src/OrdinalQuartile.ts]: Should have a lot in common with the other candlestick?

**Pull request closed without merging**